docs/spec/src/integration/spec/5-lifecycle-phases.md

```diff
@@ -24,23 +17,21 @@ EigenDA *Proxy* is used as the main entrypoint for secure dispersal and retrieva
 
 
 ### System Flow
-1. *Proxy* takes a raw [payload](./3-data-structs.md#payload) bytes on `/put` endpoint and [converts](#payload-to-blob-encoding) it into a [blob](./3-data-structs.md#blob).
-
-1. Using `latest_block_number` (lbn) number fetched from ETH RPC node, *Proxy* calls the router to get the `EigenDACertVerifier` [contract](./4-contracts.md#eigendacertverifier) address *most likely* (if using [`EigenDACertVerifierRouter`](./4-contracts.md#eigendacertverifierrouter)) to be committed to by the `reference_block_number` (rbn) returned by the EigenDA disperser.
+1. *EigenDA Client* takes a raw [payload](./3-data-structs.md#payload) bytes and [converts](#payload-to-blob-encoding) it into a [blob](./3-data-structs.md#blob).
 
-2. Using the `verifier`, *Proxy* fetches the `required_quorums` and embeds them into the [`BlobHeader`](./3-data-structs.md#blobheader) as part of the disperser request.
+1. Using `latest_block_number` (lbn) number fetched from ETH RPC node, *EigenDA Client* calls the router to get the `EigenDACertVerifier` [contract](./4-contracts.md#eigendacertverifier) address *most likely* (if using [`EigenDACertVerifierRouter`](./4-contracts.md#eigendacertverifierrouter)) to be committed to by the `reference_block_number` (rbn) returned by the EigenDA disperser.
```

Comment:
Instead of using bold `*EigenDA Client*` everywhere, should we instead use [EigenDA Client] and add a markdown link at the bottom of the page:

```
[EigenDA Client]: ../../glossary.md
```

Reply:
op does this in their docs, which is nice to always send people to the glossary

Reply:
9d35a2438b9d4e928bbab9d95f52d55815294cc7